### ISSUE TITLE
Python 3.4+ / Django 1.8+ support

### ISSUE THREAD

**Issue by eb-emilio** [markfinger/python-react#93]
Is there any plan to run travis on newer versions of Python and Django?

```
python:
  - "2.7"
  - "3.4"

env:
  - DJANGO_VERSION=1.7.8
  - DJANGO_VERSION=1.8.1
```

Code is not Python 3.7 compatible at least because it is calling `unicode` which is not available.

I volunteer to add this support in a pull request if you are interested.

**Comment by markfinger** [comment]
Sure, sounds good 👍 

I think the code should function in both py2 and py3 as the `unicode` call is wrapped in `__unicode__` which would only get called on py2. That being said maybe there's something in 3.7 that I'm not aware of 🤷

**Comment by markfinger** [comment]
Closed by #94

**Comment by markfinger** [comment]
Looks like there are some compatibility issues: https://github.com/markfinger/python-react/pull/94.

We'll need to figure out what we're aiming to support.

**Python**

I'm happy to drop py2 support. Internally we've got a few projects still running on it, but they're largely in a slow-maintenance / legacy mode.

**Django**

From https://www.djangoproject.com/download/, it looks Django 3.x and Django 2.2 are still supported. Recommend we consolidate on those.

**Comment by markfinger** [comment]
@eb-emilio any qualms if we switch the travis config to something like

```yaml
python:
  - "3.8"

env:
  - DJANGO_VERSION=2.2.16
  - DJANGO_VERSION=3.1.1
```

**Comment by eb-emilio** [comment]
@markfinger yes, that makes sense. Sorry about that. We are on the same boat on legacy apps but it sounds good to stop testing Python 2.x and Django 1.x since they don't have support anymore.
